README.md

```diff
@@ -15,6 +15,8 @@ GCP Cloud Build watches this branch.  On every push, it runs the pipeline define
 
 - `_GOARCH=amd64 _GOOS=darwin`
 - `_GOARCH=amd64 _GOOS=linux`
+- `_GOARCH=arm64 _GOOS=linux`
+- `_GOARCH=ppc64le _GOOS=linux`
```

Comment:
Do this need to be listed somewhere else? I mean you will need to configure this list of os/arch somewhere (maybe not in code though).

Reply:
Yeah I think @DirectXMan12 needs to update these vars in gcloud.